amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -118,6 +118,13 @@ vx_status WarpAffinebatchPD_Register(vx_context);
 vx_status WarpPerspectivebatchPD_Register(vx_context);
 vx_status Resizetensor_Register(vx_context);
 
+//tensor
+vx_status Brightness_Register(vx_context);
+vx_status Resize_Register(vx_context);
+vx_status CropMirrorNormalize_Register(vx_context);
+vx_status NopTensor_Register(vx_context);
```

Comment:
Do we need to mention these as Noptensor?
Please check with @shobana-mcw

In tensor branch we have changes to nop and copy for tensor
and NopBatchPD and CopyBatchPD